Gemfile.jruby-1.9.lock.release

```diff
@@ -424,7 +425,7 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       manticore (>= 0.5.4, < 1.0.0)
       stud (~> 0.0, >= 0.0.17)
-    logstash-output-file (4.1.2)
+    logstash-output-file (4.2.1)
```

Comment:
Support overwrite option for file output.

Reply:
Unless there is a compelling reason, i would suggest to not bump this minor version for the next patch.  I don't think that new functionality is expected to be included in patch versions.

Reply:
This has been our policy throughout 5.x. I'm glad to discuss changing it though. I agree it's weird in some ways.